engine/common/custom_celery_tasks/dedicated_queue_retry_task.py

```diff
@@ -1,9 +1,11 @@
-from celery import Task, shared_task
+from celery import shared_task
+
+from common.custom_celery_tasks.log_exception_on_failure_task import LogExceptionOnFailureTask
 
 RETRY_QUEUE = "retry"
 
 
-class DedicatedQueueRetryTask(Task):
+class DedicatedQueueRetryTask(LogExceptionOnFailureTask):
```

Comment:
This _should_ cause this change to take effect for _all_ of our tasks (except for the three changed engine/apps/alerts/tasks/check_escalation_finished.py which used `@shared_task` instead of `@ shared_dedicated_queue_retry_task`).

We have one other case where `@shared_task` is used, but the `base` arg is set to `CreateAlertBaseTask` which subclasses `DedicatedQueueRetryTask` (which in turn subclasses `LogExceptionOnFailureTask`)